Parsers/ASimAuthentication/Tests/Salesforce_ASimAuthentication_DataTest.csv

```diff
@@ -0,0 +1,24 @@
+﻿Result
+"(0) Error: 1 invalid value(s) (up to 10 listed) in 10 records (33.33%) for field [ActorUserIdType] of type [Enumerated]: [""SaleforceId""] (Schema:Authentication)"
+"(0) Error: 1 invalid value(s) (up to 10 listed) in 10 records (33.33%) for field [IpAddr] of type [IP Address]: [""Salesforce.com IP""] (Schema:Authentication)"
```

Comment:
Can we replace Salesforce.com IP with some internal loopback IP 127.0.0.1 or 0.0.0.0

Reply:
'Salesforce.com IP' is the IPAddress value in sample data. thats what coming in this error. This is not an actual error. We have actual IP Address also coming in sample data.
![image](https://github.com/Azure/Azure-Sentinel/assets/97222872/e219bce0-a3fb-48c2-adb3-2b6f719f2260)